app/components/Ability.tsx

```diff
@@ -40,8 +40,11 @@ export function Ability({
     setIsDragTarget(false);
   };
 
+  // Render an ability as a button only if it is meant to be draggable (i.e., not readonly)
+  const AbilityTag = readonly ? "div" : "button";
+
   return (
-    <button
+    <AbilityTag
```

Comment:
Woops, not sure why the changes from https://github.com/Sendouc/sendou.ink/pull/1057 are showing up here - please ignore these changes, I can clean them up prior to finalizing the PR